frontend/js/src/artist/ArtistPage.tsx

```diff
@@ -27,7 +27,11 @@ import {
   ListeningStats,
   popularRecordingToListen,
 } from "../album/utils";
-import type { PopularRecording, ReleaseGroup, SimilarArtist } from "../album/utils";
+import type {
```

Comment:
The are a bunch of files in the PR that only have automatic formatting changes. Ideally those should not be committed as there is no reason to add them to the PR.
If you are doing `git add .` before committing then you need to start reviewing the changes that you actually want to commit :)

Reply:
Hi! Yeah so sorry for that, I usually remove these files, but forgot about them this time. I will change it back to normal